docs/ecosystem/community.md

```diff
@@ -79,6 +79,18 @@ This isn't an exhaustive list. Want your own content here? Create a PR!
 
 ## Community Articles & Tutorials
 
+#### Ory Blog Guest Articles
+
+> You are building something with Ory or are interested in our topics in
+> general? [Reach out to us](mailto:office@ory.sh) to be published in our blog!
```

Comment:
```suggestion
Are you building something with Ory? Are you interested in Auth and security topics in general? Do you want to share your knowledge and experiences? [Reach out to us](mailto:office@ory.sh) to be published in our blog!
```

Reply:
You don't really need that "quote" formatting here, do you? I hope this sounds a bit nicer now :)